README.md

```diff
@@ -70,8 +53,8 @@ To build Go code, there are several options:
 
 - Without Go module support: `git clone` the repo manually:
 
-  ```
-  git clone https://github.com/grpc/grpc-go.git $GOPATH/src/google.golang.org/grpc
+  ```console
+  $ git clone https://github.com/grpc/grpc-go.git $GOPATH/src/google.golang.org/grpc
```

Comment:
The reason we included the prompt above was to distinguish between the command the user ran and the error message output.

Here and below it would be nice to omit it so users can copy/paste more easily.

Reply:
Done